src/si/volume_rate.rs

```diff
@@ -56,6 +56,28 @@ quantity! {
         @cubic_yoctometer_per_second: prefix!(yocto) * prefix!(yocto) * prefix!(yocto);
             "ym³/s", "cubic yoctometer per second", "cubic yoctometers per second";
 
+        @yottaliter_per_second: prefix!(milli) * prefix!(yotta); "YL/s", "yottaliter per second", "yottaliters per second";
```

Comment:
Wrap at 100 characters. Just before the plural description.

Reply:
Weird that `rustfmt` doesn't catch these.

Reply:
Ah, from the [list of limitations](https://github.com/rust-lang/rustfmt#limitations) in the `rustfmt` README:

> Macro declarations and uses (current status: some macro declarations and uses are formatted).